### DISCUSSION TITLE
dotnet client - token

### DISCUSSION THREAD

**Discussion by samjunction** [ory/.github#19] [Q&A]
How to issue oauth2 access token from dotnet client ? The Ory Hydra CLI offers a method "hydra token client", not sure which equivalent method is there in dotnet client.

**Comment by vinckr** [comment]
.Net [Documentation for API Endpoints](https://github.com/ory/sdk/tree/master/clients/hydra/dotnet#documentation-for-api-endpoints). 
I don't think there is an equivalent method for the token client command, as that is used to test a client credentials flow, you can just reproduce the flow with the above methods.
